tests/test_inbox.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+import pytest
+from flask import url_for
+from flask.testing import FlaskClient
+
+from hushline.db import db
+from hushline.model import Message, User, Username
+
+# Define test passwords
+TEST_USER_PASSWORD = os.getenv("TEST_USER_PASSWORD", "TestPassword123!")
+OTHER_USER_PASSWORD = os.getenv("OTHER_USER_PASSWORD", "OtherUserPass456!")
+
+
+@pytest.fixture()
+def setup_user_data(user: User) -> None:
```

Comment:
We already have fixtures `user_alias` and `message` that would do this. Is there something here that isn't covered by those?

Reply:
updated